src/commands/protocols/publish-paranet/sender/publish-paranet-validate-asset.command.js

```diff
@@ -27,6 +27,66 @@ class PublishParanetValidateAssetCommand extends ValidateAssetCommand {
             storeType = LOCAL_STORE_TYPES.TRIPLE,
         } = command.data;
 
+        // Validate node is in paranet
+        const paranetId = this.paranetService.constructParanetId(blockchain, contract, tokenId);
```

Comment:
tokenId and contract are from asset that are being publish,  we should use paranetUAL which is field of cachedData